docs/go/search-apis.md

```diff
@@ -6,15 +6,13 @@ sidebar_label: Search APIs
 
 ## Overview
 
-The [Go SDK Client](https://pkg.go.dev/go.temporal.io/sdk/client#Client) offers APIs for searching for and listing Workflows.
+The [Go SDK Client](https://pkg.go.dev/go.temporal.io/sdk/client#Client) offers APIs for instrumenting Workflows with additional data that can later be searched via the Web UI.
```

Comment:
Search is also available via the SDK through the List APIs:
https://github.com/temporalio/sdk-go/blob/6580cbe0aa41a8b515791f95c2c15bb37db1dab1/client/client.go#L241